air_to_breath/blocks/blocks.py

```diff
@@ -110,16 +110,30 @@ def test_method(self):
 
 class ValidateSensorsError(TestBlock):
     setup: AirToBreathSetup = BlockInput()
+    expected_values: dict = BlockInput()
+
+    CHECK_OPTIONS = [
+        'LOW',
+        'HIGH'
+    ]
 
-    # sensors: list = BlockInput()
+    def validate(self, sensor, expected_value, what_to_check='LOW'):
+        if what_to_check == 'LOW':
+            template = SENSORS[sensor].LOW_ERROR_TEMPLATE
 
-    def validate(self, sensor):
-        template = getattr(self.setup, sensor).ERROR_TEMPLATE
+        elif what_to_check == 'HIGH':
+            template = SENSORS[sensor].HIGH_ERROR_TEMPLATE

```

Comment:
Your code will crash if what_to_check isn't high or low, as no template defined.
How about sensor_error or something like this instead of what_to_check?